schema.json

```diff
@@ -2621,125 +2619,125 @@
-              "id": "paymentbuttons-paypal-color",
-              "force_reload": true,
-              "options": [
-                {
-                  "value": "gold",
-                  "label": "Gold"
-                },
-                {
-                  "value": "blue",
-                  "label": "Blue"
-                },
-                {
-                  "value": "silver",
-                  "label": "Silver"
-                },
-                {
-                  "value": "black",
-                  "label": "Black"
-                }
-              ]
-          },
-          {
-              "type": "select",
-              "label": "Shape",
-              "id": "paymentbuttons-paypal-shape",
-              "force_reload": true,
-              "options": [
-                {
-                  "value": "pill",
-                  "label": "Pill"
-                },
-                {
-                  "value": "rect",
-                  "label": "Rectangle"
-                }
-              ]
-          },
-          {
-              "type": "select",
-              "label": "Size",
-              "id": "paymentbuttons-paypal-size",
-              "force_reload": true,
-              "options": [
-                {
-                  "value": "small",
-                  "label": "Small"
-                },
-                {
-                  "value": "medium",
-                  "label": "Medium"
-                },
-                {
-                  "value": "large",
-                  "label": "Large"
-                },
-                {
-                  "value": "responsive",
-                  "label": "Responsive"
-                }
-              ]
-          },
-          {
-              "type": "select",
-              "label": "Display Label",
-              "id": "paymentbuttons-paypal-label",
-              "force_reload": true,
-              "options": [
-                {
-                  "value": "checkout",
-                  "label": "Paypal Checkout"
-                },
-                {
-                  "value": "pay",
-                  "label": "Pay with Paypal"
-                },
-                {
-                  "value": "buynow",
-                  "label": "Buy Now"
-                },
-                {
-                  "value": "paypal",
-                  "label": "Paypal"
-                }
-              ]
-          },
-          {
-              "type": "checkbox",
-              "label": "Display the Paypal Tagline",
-              "force_reload": true,
-              "id": "paymentbuttons-paypal-tagline"
-          },
-          {
-              "type": "checkbox",
-              "label": "Display the Funding Icons",
-              "force_reload": true,
-              "id": "paymentbuttons-paypal-fundingicons"
+    "name": "Payment Buttons",
+    "enable": "smartButtons",
+    "settings": [
+      {
+        "type": "checkbox",
+        "label": "Show Paypal tagline",
```

Comment:
Should be `PayPal` with two capital `P`s.

Reply:
:+1: Thank you